meta-dts-distro/recipes-support/mei-amt-check/files/0001-Makefile-removed-setting-CC-variable.patch

```diff
@@ -15,8 +15,7 @@ index c98d181..e1700dc 100644
 -CC := $(CROSS_COMPILE)gcc
 -
  PROGS := mei-amt-check
- 
+
  all: $(PROGS)
--- 
+--
```

Comment:
Modification of patches may break them, we can consider skipping some rules for patch files. @DaniilKl had some ideas on this IIRC.

Reply:
@PLangowski To exclude `.patch` files from being `fixed` by this hook

```suggestion
      - id: trailing-whitespace
        exclude: .*\.patch$
```

You can also add `exclude` at the top level to exclude patches from being checked by pre-commit at all

Reply:
I restored the patches to their original form https://github.com/Dasharo/meta-dts/pull/129/commits/8725595a288f981ff2a1a8b78e397d015b39c149